aws/table_aws_cloudwatch_log_group.go

```diff
@@ -176,19 +189,20 @@ func listCloudwatchLogGroups(ctx context.Context, d *plugin.QueryData, _ *plugin
 //// HYDRATE FUNCTIONS
 
 func getCloudwatchLogGroup(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	name := d.EqualsQualString("name")
```

Comment:
@ParthaI shouldn't we be adding an additional check for the `region` column as well?

If a user does not pass in the region along with the name, then we should return `nil`.

Reply:
@misraved, in accordance with our design, we utilize the [SupportedRegionMatrix](https://github.com/turbot/steampipe-plugin-aws/pull/1976/files#diff-ad2cd4c57b9003b968bc7b23321bcd5cd1e920fd1a51f0bc9e663462126509a9R48). 

Here's how it works:
- If a user queries without specifying the `region` but includes the `name`, we will perform a list API call. This approach ensures that there will be no errors such as 'key column is not globally unique'.
- Conversely, if the `region` value is specified along with the `name`, a Get API call will be executed, but only for the specified `region`.


Note:  We can have multiple LogGroups with same name but in different region. Therefore, our approach ensures that users receive actual results rather than an empty row in such cases.